### PR TITLE
[QA] Window size test features

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers
@flopezag @geonexus 

#### Description
This PR resolves the feature *CLAUDIA-4733*. This code implements some acceptance tests for *CLAUDIA-4259*. Main features included:
- RabbitMQ Consumer and Publisher for testing purposes.
- Test cases for FACTS' window size behavior

#### Testing
Locally.

### CODE REVIEW

File: tests/acceptance/features/component/info.feature [hunk 1]
```diff
@@ -9,5 +9,5 @@ Feature: Get 'fiware-facts' service info
   Scenario: Get info from Fiware-Facts service.
     Given the fiware-facts service properly deployed
     When  I request the service info
-    Then  I receive a HTTP "200" response code
+    Then  the HTTP "200" is returned
```
Comment (flopezag) [inline tests/acceptance/features/component/info.feature]: Honestly, I prefer the step:

I receive a HTTP response with response code "200"

I do not know if it means a lot of changes.

File: tests/acceptance/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
 PyHamcrest>=1.8.3
-git+https://github.com/qaenablers/python-qautils.git@develop#egg=python-qautils
+git+https://github.com/qaenablers/python-qautils.git@v1.2.0
 python-keystoneclient==1.3.0
+pika==0.10.0
```
Comment (geonexus) [inline tests/acceptance/requirements.txt]: oslo libs are not needed?

**Comment by flopezag** [comment]
LGTM

**Comment by geonexus** [comment]
LGTM

**Pull request merged**